### PR TITLE
iot unallocated rewards

### PR DESCRIPTION
**Pull request by andymck**

### CODE REVIEW

File: src/service/poc_lora.proto [hunk 1]
```diff
@@ -237,6 +237,21 @@ message operational_reward {
...
+  unallocated_reward_type_oracle = 2;
+  unallocated_reward_type_data = 3;
+}
+
+message unallocated_reward {
+  // the reward type representing a reward category to which an unallocated
+  // amount exists
+  unallocated_reward_type reward_type = 1;
+  /// Amount in bones unallocated
+  uint64 amount = 2;
```
Comment (jeffgrunewald) [inline src/service/poc_lora.proto:252]: these are always hnt bones? no need to denote the token type of the unallocated amount?

Comment (andymck) [reply src/service/poc_lora.proto:252]: all reward outputs are in bones, yes.  I am not sure what you are asking on the second question

Comment (jeffgrunewald) [reply src/service/poc_lora.proto:252]: asking specifically if this is _HNT_ bones, _IoT_ bones, or if it could be either or ever

Comment (andymck) [reply src/service/poc_lora.proto:252]: gotcha now, these are iot bones and only iot bones...i'll update the comment to include 'iot'

Comment (andymck) [reply src/service/poc_lora.proto:252]: fixed here 96d71c04056ddc741f5e89aef6ce34fe3ce4bb80

**Review by jeffgrunewald — APPROVED** [review]

**Pull request merged**